doc/attr.xml

```diff
@@ -1310,3 +1310,42 @@ fail]]></Example>
   </Description>
 </ManSection>
 <#/GAPDoc>
+  
+<#GAPDoc Label="HamiltonianPath">
+<ManSection>
+  <Prop Name="HamiltonianPath" Arg="digraph"/>
+  <Returns>A list or <K>fail</K>.</Returns>
+  <Description>
+    Returns a Hamiltonian path if one exists, <K>fail</K> if not.
+
+    A <E>Hamiltonian path</E> is a directed circuit visiting
+    each vertex exactly once. See Section <Ref Subsect="Definitions"
+     Style="Number" /> for the definition of a directed circuit.
+    Note the empty digraphs on 0 and 1 vertices are considered
+    Hamiltonian.<P/>
+
+    The method used in this operation has the worst case complexity as
```

Comment:
It's not currently an `operation` - so make sure this agrees with however it is declared.